release/src/module_dependencies/main.py

```diff
@@ -159,8 +162,6 @@ def calculateLevels(moduleNameList, moduleDetailsJSON):
 
 # Updates the stdlib_modules.JSON file with dependents of each standard library module
 def updateJSONFile(updatedJSON):
-    updatedJSON['modules'].sort(key=lambda s: s['level'])
```

Comment:
Are we removing the sorting part?

Reply:
I've moved it to the main method as I'm using it to update the README file as well